src/tools/genpolicy/update_policy_samples.py

```diff
@@ -0,0 +1,95 @@
+    "deployment/deployment-azure-vote-front.yaml",
+    "job/test-job.yaml",
+    "kubernetes/conformance/conformance-e2e.yaml",
+    "kubernetes/conformance/csi-hostpath-plugin.yaml",
+    "kubernetes/conformance/csi-hostpath-testing.yaml",
+    "kubernetes/conformance/etcd-statefulset.yaml",
+    "kubernetes/conformance/hello-populator-deploy.yaml",
+    "kubernetes/conformance/netexecrc.yaml",
+    "kubernetes/conformance2/ingress-http-rc.yaml",
+    "kubernetes/conformance2/ingress-http2-rc.yaml",
+    "kubernetes/conformance2/ingress-multiple-certs-rc.yaml",
+    "kubernetes/conformance2/ingress-nginx-rc.yaml",
+    "kubernetes/conformance2/ingress-static-ip-rc.yaml",
+    "kubernetes/fixtures/appsv1deployment.yaml",
+    "kubernetes/fixtures/daemon.yaml",
+    "kubernetes/fixtures/deploy-clientside.yaml",
+    "kubernetes/fixtures/job.yaml",
+    "kubernetes/fixtures/multi-resource-yaml.yaml",
+    "kubernetes/fixtures/rc-lastapplied.yaml",
+    "kubernetes/fixtures/rc-noexist.yaml",
+    "kubernetes/fixtures/replication.yaml",
+    "kubernetes/fixtures2/rc-service.yaml",
+    "kubernetes/fixtures2/valid-pod.yaml",
+    "kubernetes/incomplete-init/cassandra-statefulset.yaml",
+    "kubernetes/incomplete-init/controller.yaml",
+    "kubernetes/incomplete-init/cockroachdb-statefulset.yaml",
+    "kubernetes/incomplete-init/node_ds.yaml",
+    "pod/pod-exec.yaml",
+    "pod/pod-one-container.yaml",
+    "pod/pod-persistent-volumes.yaml",
+    "pod/pod-same-containers.yaml",
+    "pod/pod-spark.yaml",
+    "pod/pod-three-containers.yaml",
+    "replica-set/replica-busy.yaml",
+    "secrets/azure-file-secrets.yaml",
+    "stateful-set/web.yaml",
+]
+
+silently_ignored_yaml_files=[
+    "webhook/webhook-pod1.yaml",
+    "webhook/webhook-pod2.yaml",
+    "webhook/webhook-pod3.yaml",
+    "webhook/webhook-pod4.yaml",
+    "webhook/webhook-pod5.yaml",
+    "webhook/webhook-pod6.yaml",
+    "webhook/webhook-pod7.yaml",
+    "webhook2/webhook-pod8.yaml",
+    "webhook2/webhook-pod9.yaml",
+    "webhook2/webhook-pod10.yaml",
+    "webhook2/webhook-pod11.yaml",
+    "webhook2/webhook-pod12.yaml",
+    "webhook2/webhook-pod13.yaml",
+]
+
+no_policy_yaml_files=[
+    "kubernetes/fixtures/limits.yaml",
+    "kubernetes/fixtures/namespace.yaml",
+    "kubernetes/fixtures/quota.yaml",
+]
+
+file_base_path = "../../agent/samples/policy/yaml"
+
+# check we can access all files we are about to update
+for file in yaml_files + silently_ignored_yaml_files + no_policy_yaml_files:
+    filepath = os.path.join(file_base_path, file)
+    if not os.path.exists(filepath):
+        print(f"filepath does not exists: {filepath}")
+
+# build tool
+subprocess.run(["cargo build"], stdout=sys.stdout, stderr=sys.stderr, shell=True)
+
+# update files
+genpolicy_path = "target/debug/genpolicy"
+for file in yaml_files:
+    filepath = os.path.join(file_base_path, file)
+    subprocess.run([genpolicy_path, "-y", filepath], stdout=sys.stdout, stderr=sys.stderr,)
```

Comment:
Would be good to check if the app returned 0/success.

Reply:
Good call. Added a check to the return code. Thanks!